jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: classic-docs
+title: "Insights metrics glossary"
+short-title: "Insights metrics glossary"
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
+
+This document provides definitions for all the metrics in CircleCI Insights. You
+can review metrics in the Insights dashboard as well as the Insights API.
+
+* TOC
+{:toc}
+
+
+## General metrics 
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 11. You can fix by inserting `{: #general-metrics }` to the next line